src/gui/newwizard/setupwizardwindow.cpp

```diff
@@ -32,12 +53,29 @@ SetupWizardWindow::SetupWizardWindow(QWidget *parent)
         emit paginationEntryClicked(_ui->pagination->activePageIndex(), clickedPageIndex);
     });
 
+    resize(ocApp()->gui()->settingsDialog()->sizeHintForChild());
+
+    loadStylesheet();
+
     _ui->transitionProgressIndicator->setFixedSize(32, 32);
+    _ui->transitionProgressIndicator->setColor(Theme::instance()->wizardHeaderTitleColor());
 
     // handle user pressing enter/return key
     installEventFilter(this);
 }
 
+void SetupWizardWindow::loadStylesheet()
+{
+    QString path = QStringLiteral(":/client/resources/wizard/style.qss");
+
+    QFile file(path);
+    Q_ASSERT(file.exists());
+    Q_ASSERT(file.open(QIODevice::ReadOnly));
```

Comment:
That way the fill will only be opened in debug builds.
OC_ENSURE?

Reply:
Oh yes! 
How about oldschool without _"I have to look up what OC_ENSURE really does"_ for Newcomers and CTOs using `if( ... )`?

Reply:
but our OC_ENSURE is an if + an assert :D